llm_demo/orchestrator/langgraph/tool_node.py

```diff
@@ -77,8 +78,9 @@ def run_one(call: ToolCall, user_id_token: Optional[str]):
             args = copy.copy(call["args"]) or {}
             args["user_id_token"] = user_id_token
             output = self.tools_by_name[call["name"]].invoke(args, config)
+            tool_call_id = call.get("id") or str(uuid.uuid4())
```

Comment:
Why would there not be an id? Can we add a comment on why it's ok to just generate a new UUID?

Reply:
Since ToolCall.id is optional, there's a possibility that there's no id. 
ToolMessage.id is required.